src/module/actor/sheet/popups/ability-builder.ts

```diff
@@ -3,14 +3,15 @@ import { Abilities } from "@actor/creature/data";
 import { AbilityString } from "@actor/types";
 import { ABILITY_ABBREVIATIONS } from "@actor/values";
 import { AncestryPF2e, BackgroundPF2e, ClassPF2e } from "@item";
+import { ActorPF2e } from "@actor/index";
 
-export class AbilityBuilderPopup extends Application {
+export class AbilityBuilderPopup extends FormApplication<ActorPF2e> {
```

Comment:
There's no point in making this a `FormApplication` only to later prevent it from being submitted.